doc/release/upcoming_changes/22229.expired.rst

```diff
@@ -0,0 +1 @@
+The style keyword argument has been removed from np.array2string.
```

Comment:
```suggestion
* The style keyword argument has been removed from `numpy.array2string`.
```
We need the `*` to make it a nice bullet point list in the final release notes (because NumPy allows mixing bullets and titles).